pandas/_config/config.py

```diff
@@ -158,8 +158,8 @@ def _set_option(*args, **kwargs) -> None:
             o.validator(v)
 
         # walk the nested dict
-        root, k = _get_root(key)
-        root[k] = v
+        root, k_root = _get_root(key)
+        root[k_root] = v
```

Comment:
I'm not familiar enough with this part of the code to tell if this new name makes sense, I'll take another look next week

(or, if you've already understood it, feel free to explain)

Reply:
I see this method is called from the option context, for example by running:
```
from pandas import option_context
with option_context('display.max_rows', 10, 'display.max_columns', 5):
    pass
```
The `args*` in the call of `option_context` will be passed to _set_option two at the time. From what I see the `_get_root(key)` seems to split the first element passed (in my example above could be `"display.max_rows"`) by dot. Then returns them in a root and key format. From what I understood the `k` element return is associated to the root, that's why I called it `k_root`.

Let me know if I wasn't clear or I made a mistake in understanding the function.

Reply:
ok thanks for explaining!